static/usage/select/typeahead/angular/example_component_ts.md

```diff
@@ -0,0 +1,111 @@
+```ts
+import { Component, ViewChild } from '@angular/core';
+import { IonModal } from '@ionic/angular';
+
+interface Fruit {
+  text: string;
+  value: string;
+}
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent {
+  @ViewChild('modal', { static: true }) modal!: IonModal;
+  
+  filteredFruits: Fruit[] = [...this.fruits];
```

Comment:
This needs to be moved under the `fruits` declaration to avoid a compilation error:
![error](https://i.gyazo.com/3d1be029a5c2678ed239902652cccadd.png)